packages/token-bridge-sdk/src/hooks/useArbTokenBridge.ts

```diff
@@ -235,19 +277,27 @@ export const useArbTokenBridge = (
   async function withdrawEth({
     amount,
     l2Signer,
-    txLifecycle
+    txLifecycle,
+    destinationAddress
   }: {
     amount: BigNumber
     l2Signer: Signer
     txLifecycle?: L2ContractCallTransactionLifecycle
+    destinationAddress?: string
   }) {
     try {
       const ethBridger = await EthBridger.fromProvider(l2.provider)
 
+      verifyDestinationAddressOrThrow({
+        provider: l2Signer.provider,
+        destinationAddress,
+        isDeposit: false
+      })
+
       const tx = await ethBridger.withdraw({
         amount,
         l2Signer,
-        destinationAddress: walletAddress,
+        destinationAddress: destinationAddress ?? walletAddress,
```

Comment:
do we want to allow empty destination address?
otherwise we should use `||`

Reply:
We have multiple checks in place not to allow an empty address to get here but I will change to `||` as a precaution